html.proto

```diff
@@ -0,0 +1,12 @@
+syntax = "proto3";
+
+package pbtypes;
+
+import "github.com/gogo/protobuf/gogoproto/gogo.proto";
+
+// HTML is a type which marshals into {__html: "html code here"} to designate
+// that this value is sanitized HTML code, see
+// https://facebook.github.io/react/tips/dangerously-set-inner-html.html
+message HTML {
+  string HTML = 1 [(gogoproto.jsontag) = "__html"];
```

Comment:
Minor, I just noticed this seems to use 2 spaces for indentations, while all other .proto files we have use a single tab for indentation. /cc @neelance